delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/indexing/ElasticSearchDefaultViewsResetter.scala

```diff
@@ -92,15 +93,15 @@ object ElasticSearchDefaultViewsResetter {
       override def resetDefaultViews: IO[Unit] =
         resetTrigger.flatMap { triggered =>
           IO.whenA(triggered) {
-            views.compile.toList
-              .flatMap { _.traverse { resetView } }
-              .flatMap { _ => logger.info("Completed resetting default elasticsearch views.") }
+            deleteDatabaseEntries >>
```

Comment:
So was the fact that ES indices were deleted before the entries also an issue?

Reply:
I think it is a combination of that and the fact that we were reading from replicas when recreating the views yes.
Reducing the number of transactions is not a bad thing anyway, it will also help the WAL to get replicated.